next-tavla/src/Board/scenarios/Table/components/Situations/index.tsx

```diff
@@ -1,17 +1,30 @@
 import { Situation } from '../Situation'
 import { useNonNullContext } from 'hooks/useNonNullContext'
 import { DepartureContext } from '../../contexts'
+import { useEffect, useState } from 'react'
 
 function Situations() {
     const departure = useNonNullContext(DepartureContext)
 
+    const numberOfSituations = departure.situations.length
+    const [index, setIndex] = useState(0)
+
+    useEffect(() => {
+        if (numberOfSituations <= 1) {
+            return
+        }
+        const interval = setInterval(
+            () => setIndex((i) => (i + 1) % numberOfSituations),
+            5000,
+        )
+        return () => clearInterval(interval)
+    }, [numberOfSituations])
+
     return (
         <td>
-            <div>
-                {departure.situations.map((situation) => (
-                    <Situation key={situation.id} situation={situation} />
-                ))}
-            </div>
+            {numberOfSituations ? (
+                <Situation situation={departure.situations[index]} />
+            ) : null}
```

Comment:
Jeg tror vi bør stokke litt om på denne, fordi om det blir færre avvik så kan det hende indexen er out of bounds i fem sekunder før den blir oppdatert. Kan vi gjøre modulo utenfor useEffecten istedet?